content/doc/book/installing/_master-agent-machines-setup-windows.adoc

```diff
@@ -0,0 +1,61 @@
+[[master-agent-setup-windows]]
+== Windows Agents
+
+Note: There is no need to install Jenkins on the agent machine.
+
+. On your master machine go to *Manage Jenkins* > *Manage Nodes*. +
+image:/images/installing/ManageNodes.png[Manage nodes,width=80%]
+
+. *New Node*
+.. *Enter Node Name*
+.. *Select Permanent Agent*
+.. Press *OK*. +
+image:/images/installing/NewNode.png[New node,width=40%]
+
+. Fill out the following:
+.. Set a *number of executors*
+... (one or more) as needed.
+.. Set a *Remote FS Root*
+... a home directory for the master on the agent machine.
+... For a _Windows agent_, use something like: "C:\Jenkins\"
+.. Select the appropriate *Usage* setting:
+... For an additional worker: _Utilize this node as much as possible_
+... For specialized jobs: _Leave this machine for tied jobs only_
+.. *Launch Method*:
+... An easy way to control a Windows agent is by using _Launch agent via Java Web Start_  (Recommended for Windows)
+... TODO: add steps for other methods.
+.. *Availability*
+... _Keep this agent online as much as possible_
+... TODO: add details for each option.
+.. Press *Save*. +
+image:/images/installing/setup06.png[Save Jenkins agent definition,width=80%]
+. Now you need to connect your agent machine to the master using the following steps.
+.. Open a browser on the **agent*** machine* and go to the *Jenkins master server* url \http://yourjenkinsmaster:8080/ .
+.. Go to *Manage Jenkins* > *Manage Nodes*,
+... Click on the newly created agent machine. You will need to login as someone that has the "Connect" Agent permission if you have configured global security.
+.. Click on the *Launch* button to launch agent from browser on agent. +
+image:/images/installing/setup03.png[Connect Jenkins agent,width=90%]
+.. Run the program. +
+image:/images/installing/setup04.png[Run the program] +
+If Windows asks you to choose a program, use `+$JAVA_HOME\bin\javaws.exe+` (the Java Web Start Launcher).
+.. If you encounter connection issue, then you could enlarge the popup windows to see the master *port used* and check your network configuration (firewall, port forward, ...).
+image:/images/installing/connect.jpg[Jenkins agent Windows dialog,width=70%] +
+Note that Jenkins chooses a random, high-number port. If you want Jenkins to use a static port, go to "Manage Jenkins" > "Configure Global Security" and choose the port number in the "TCP port for JNLP agents" box.
+.. If the port is open, the agent still can't connect, and your Jenkins instance is served securely over SSL/HTTPS, assure that you're running a recent Java version so that it includes the correct Java Cryptography Extension (JCE).
+.. Now you should see the agent machine connected under *Nodes*. +
+image:/images/installing/Node.png[Connected node, width=50%]
+
+.  If you want the service to run on start-up of the agent machine do the following (Windows only directions):
```

Comment:
No need to say "Windows only directions" in a section that is clearly dedicated to Windows
```suggestion
.  If you want the service to run on start-up of the agent machine do the following:
```